packages/components/src/InputNumber/InputNumber.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { cleanup, fireEvent, render } from "@testing-library/react";
+import { InputNumber } from ".";
+
+afterEach(cleanup);
+
+it("renders an input type number", () => {
+  const tree = renderer.create(<InputNumber value="123" />).toJSON();
+  expect(tree).toMatchInlineSnapshot(`
+    <div
+      className="wrapper"
+      style={
+        Object {
+          "--formField-maxLength": undefined,
```

Comment:
Any idea how this is getting set to undefined?

Reply:
because it ain't specified. I think the tests still renders undefined stuffs